apps/mark-scan/frontend/src/pages/accessible_controller_diagnostic_screen.tsx

```diff
@@ -1,15 +1,42 @@
 import { useEffect, useState } from 'react';
 import { Button, Font, H1, Main, P, Screen } from '@votingworks/ui';
 import { DateTime } from 'luxon';
+import styled from 'styled-components';
 import { ScreenReader } from '../config/types';
 import {
   DiagnosticScreenHeader,
   StepContainer,
-  StepInnerContainer,
 } from './diagnostic_screen_components';
 
 type ButtonName = 'Up' | 'Down' | 'Left' | 'Right' | 'Select';
 
+const StepInnerContainer = styled.div`
```

Comment:
I had factored this out earlier for use in the PAT flow, but since the designs have diverged it no longer made sense to share styles. This just adds this styled element back to this file; no diff from `main`.